types/adm-zip/index.d.ts

```diff
@@ -204,7 +208,7 @@ declare namespace AdmZip {
         /**
          * Get the header associated with this ZipEntry.
          */
-        header: Buffer;
+        readonly header: EntryHeader;
```

Comment:
> You should update the version in `index.d.ts`:
> ```ts
> // Type definitions for adm-zip 0.5
> ```

Actually these are still for 0.4 - I did not include any of the API changes in 0.5.

Reply:
> Why `readonly`? There's a setter for this field: https://github.com/cthackers/adm-zip/blob/master/zipEntry.js#L264

The getter returns `EntryHeader`, but the setter takes in a `Buffer`, not `EntryHeader`. TypeScript doesn't yet support getter and setter having different types (https://github.com/microsoft/TypeScript/issues/2521), so the alternative without `readonly` would be something like `header: EntryHeader | Buffer`, I guess. (But I think reading this field would be much, much more common than setting it, so that would require all readers to typecast)

Reply:
sounds good to me